bin.src/generateCoefficients.py

```diff
@@ -11,114 +13,132 @@
                                      " for a set of orbits, over a given timespan.")
     parser.add_argument("--orbitFile", type=str, default=None,
                         help="File containing the orbits.")
+    parser.add_argument("--objStart", type=int, default=None,
+                        help="Start coefficient generation at object number 'objStart'. "
+                        "Default None - start at 0.")
+    parser.add_argument("--objEnd", type=int, default=None,
+                        help="End coefficient generation at object number 'objEnd'. "
+                        "Default None - finish at last object.")
+    parser.add_argument("--nObj", type=int, default=None,
+                        help="If specified, then orbitFile is processed and "
+                        "written to disk in chunks of 'nObj'.")
     parser.add_argument("--tStart", type=float, default=None,
                         help="Start of timespan to generate coefficients.")
     parser.add_argument("--tEnd", type=float, default=None,
                         help="End of timespan to generate coefficients.")
     parser.add_argument("--tSpan", type=float, default=None,
-                        help="Length of timespan (specify this OR tEnd).")
+                        help="Chunk total time into subsets of tSpan (multiple output files)."
+                        " If one of tSpan or tEnd is missing, assumes one chunk was meant.")
     parser.add_argument("--skyTol", type=float, default=2.5,
                         help="Sky tolerance for position residuals; default 2.5 mas")
-    parser.add_argument("--nObj", type=int, default=None,
-                        help="If defined, then only nObj coefficients are calculated "
-                        "and written to disk at once.")
     parser.add_argument("--length", type=float, default=None,
```

Comment:
`"--length"` docstring:  When I read "Chebyshev polynomial length" I think of the polynomial degree. For example, what would you expect from: len(polynomial)? Are these the time "segment lengths"  in units of days? If so,  I would replace the docstring with "Segment length (in days) for each polynomial fit: time span of a single row in coefficients file. (Will self-determine from skyTol and nCoeff if not given.)"